src/sonic-config-engine/minigraph.py

```diff
@@ -392,8 +394,10 @@ def parse_dpg(dpg, hname):
                     # later after the rest of the minigraph has been parsed.
                     acl_intfs = pc_intfs[:]
                     for panel_port in port_alias_map.values():
-                        if panel_port not in intfs_inpc:
-                            acl_intfs.append(panel_port)
+                        #because of port_alias_asic_map we can have duplicate in  port_alias_map
```

Comment:
Add space after `#` and remove extra space before `port_alias_map`

Reply:
@jleveque Done

Reply:
Extra space is still present

Reply:
@ Sorry updated now

---
In reply to: [421707430](https://github.com/Azure/sonic-buildimage/pull/4491#discussion_r421707430) [](ancestors = 421707430)